packages/shared/src/medical/fhir/observations.ts

```diff
@@ -0,0 +1,58 @@
+import { Observation } from "@medplum/fhirtypes";
+
+// Not sure the names of the codes are correct,
+// but they're related and should be grouped together regardless
```

Comment:
Is this a comment just for the PR / reviewers? I think we should not leave it on the code as it's confusing - when I read this, I'm less confident in that code.

I suggest we confirm on Slack if it's not clear and/or we're not sure how to go about it, and remove the comment.

Reply:
sorry this was actually a note to myself i forgot to remove.